tools/rosbag/package.xml

```diff
@@ -20,7 +20,11 @@
   <author>Jeremy Leibs</author>
   <author>James Bowman</author>
 
-  <depend>boost</depend>
+  <depend>libboost-date-time-dev</depend>
+  <depend>libboost-filesystem-dev</depend>
+  <depend>libboost-program-options-dev</depend>
+  <depend>libboost-regex-dev</depend>
+  <depend>libboost-thread-dev</depend>
```

Comment:
Could we split this into `exec_depend` and the various `build*_depend`?

Did you not do this on purpose?

Reply:
Purposeful :grimacing: , currently I just replaced the keys without changing the scope of the tags.

I figured that when splitting the `depend`s tags, we should decide which to put in `build_depend` vs `exec_depend` vs `build_export_depend` which would require a closer audit of each package rather than just duplicating them all in build and exec depends.

It could be done in this PR though, it was just out of scope of my first pass